### ISSUE TITLE
[FEATURE REQUEST] Hide Tooltip with Shift (or other hotkey)

### ISSUE THREAD

**Issue by jessev14** [bmarian/token-tooltip-alt#49]
Sometimes the tooltip covers other tokens or important information on the canvas. It would be convenient to be able to temporarily disable/enable the tooltip on command.

One possible solution could be to set the Shift key (or a user defined hotkey) to temporarily disable the tooltip while pressed.

Another method could be to introduce a setting in which the tooltip is disabled by default, but appears when you hold Shift.

@enso0361

**Comment by buttonpushertv** [comment]
I agree with the suggestion of adding an option that will *only* show the tooltip if you press a modifier key. In fact, that's what I had originally thought the setting "Show on <ALT>" meant - that you would only see the tooltip *if * you were holding ALT down as you hovered over it.

I'd like to have access to the tooltip info, but not all the time. I like the idea of hovering plus a modifier to show it.

**Comment by Fraxinus1753** [comment]
I came here to suggest the same. This is a great module, and it would be made even better with the option to conditionally activate show on hover (or deactivate it completely, I would be fine with just pressing alt to show tooltips for all tokens and that's it).
